diff --git a/dist/cjs/core.d.cts b/dist/cjs/core.d.cts
index 5e92b2df45fbccc86257b91c3f26a77d5da00fb5..918e8b11d4ab3df07f0a2ac1badb23c8991b5428 100644
--- a/dist/cjs/core.d.cts
+++ b/dist/cjs/core.d.cts
@@ -1,6 +1,6 @@
 import type { SwaggerTransform, SwaggerTransformObject } from "@fastify/swagger";
 import type { FastifyPluginAsync, FastifyPluginCallback, FastifyPluginOptions, FastifySchema, FastifySchemaCompiler, FastifyTypeProvider, RawServerBase, RawServerDefault } from "fastify";
-import type { FastifySerializerCompiler } from "fastify/types/schema";
+import type { FastifySerializerCompiler } from "fastify";
 import type { $ZodRegistry, input, output } from "zod/v4/core";
 import { $ZodType } from "zod/v4/core";
 import { type ZodToJsonConfig } from "../cjs/zod-to-json.cjs";
diff --git a/dist/cjs/errors.d.cts b/dist/cjs/errors.d.cts
index f624af896ea71830160bca1e3518e591850d7dd0..de4cda537c598bff271a6aaaec73d31d8f98d474 100644
--- a/dist/cjs/errors.d.cts
+++ b/dist/cjs/errors.d.cts
@@ -1,6 +1,6 @@
 import { type FastifyErrorConstructor } from "@fastify/error";
 import type { FastifyError } from "fastify";
-import type { FastifySchemaValidationError } from "fastify/types/schema";
+import type { FastifySchemaValidationError } from "fastify";
 import type { $ZodError } from "zod/v4/core";
 export declare const InvalidSchemaError: FastifyErrorConstructor<{
 	code: string;
diff --git a/dist/esm/core.d.ts b/dist/esm/core.d.ts
index fbc716fdcb76ae45219c26261cfb908198d46149..47eb44ed92b0882a8087dac16ad05274659c7d67 100644
--- a/dist/esm/core.d.ts
+++ b/dist/esm/core.d.ts
@@ -1,6 +1,6 @@
 import type { SwaggerTransform, SwaggerTransformObject } from "@fastify/swagger";
 import type { FastifyPluginAsync, FastifyPluginCallback, FastifyPluginOptions, FastifySchema, FastifySchemaCompiler, FastifyTypeProvider, RawServerBase, RawServerDefault } from "fastify";
-import type { FastifySerializerCompiler } from "fastify/types/schema";
+import type { FastifySerializerCompiler } from "fastify";
 import type { $ZodRegistry, input, output } from "zod/v4/core";
 import { $ZodType } from "zod/v4/core";
 import { type ZodToJsonConfig } from "../esm/zod-to-json.js";
diff --git a/dist/esm/errors.d.ts b/dist/esm/errors.d.ts
index f624af896ea71830160bca1e3518e591850d7dd0..de4cda537c598bff271a6aaaec73d31d8f98d474 100644
--- a/dist/esm/errors.d.ts
+++ b/dist/esm/errors.d.ts
@@ -1,6 +1,6 @@
 import { type FastifyErrorConstructor } from "@fastify/error";
 import type { FastifyError } from "fastify";
-import type { FastifySchemaValidationError } from "fastify/types/schema";
+import type { FastifySchemaValidationError } from "fastify";
 import type { $ZodError } from "zod/v4/core";
 export declare const InvalidSchemaError: FastifyErrorConstructor<{
 	code: string;
